code/models/models.js

```diff
@@ -47,16 +47,45 @@ exports.selectOneArticle = (request) => {
 exports.selectCommentsForArticle = (request) => {
   const articleID = request.params.articles_id;
   if (!Number.isInteger(+articleID)) {
```

Comment:
This is fine, I think that if you let the invalid id go to the database it would error anyway, and you could handle that error probably by checking the code that psql gives you, your messages might not be as nice as you have done here but it would mean that you're code was a little leaner